common/txmgr/fee/models.go

```diff
@@ -0,0 +1,22 @@
+package fee // renamed Fee from fee for chain agnostic purposes
```

Comment:
This comment is suitable for as a PR comment, not inside the code

Reply:
got it! thank you